thirtyinch/build.gradle

```diff
@@ -36,14 +36,15 @@ configurations {
 }
 
 dependencies {
-    compileOnly "com.android.support:appcompat-v7:$supportLibraryVersion"
+    compileOnly "androidx.appcompat:appcompat:$androidXAppCompatVersion"
+    compileOnly "androidx.annotation:annotation:$androidXAnnotationVersion"
 
-    testCompileOnly "com.android.support:support-annotations:$supportLibraryVersion"
+    testCompileOnly "androidx.annotation:annotation:$androidXAnnotationVersion"
```

Comment:
Didn't we say, it might makes more sense to use `implementation` here?

Reply:
Yep. But I just wanted to "migrate to androidx" and not "update all the things" 🙃